index.html

```diff
@@ -1643,21 +1643,21 @@ <h4>Wallis Integrals</h4>
                         <p>\[then,\,\,\,If\,n\,is\,even,\,\,\displaystyle{ W_{2p}=\frac{2p-1}{2p} \cdot \frac{2p-3}{2p-2} \cdots \frac{1}{2} W_0 = \frac{(2p-1)!!}{(2p)!!} \cdot \frac{\pi}{2} = \frac{(2p)!}{2^{2p} (p!)^2} \cdot \frac{\pi}{2}, }\]</p>
                         <p>\[If\,n\,is\,odd,\,\,\displaystyle{ W_{2p}=\frac{2p-1}{2p} \cdot \frac{2p-3}{2p-2} \cdots \frac{1}{2} W_0 = \frac{(2p-1)!!}{(2p)!!} \cdot \frac{\pi}{2} = \frac{(2p)!}{2^{2p} (p!)^2} \cdot \frac{\pi}{2}. }\]<p>
 
-<<<<<<< HEAD
+<!--<<<<<<< HEAD
 
 =======
->>>>>>> 2a361fd1b78219385ebf356d98fd63716e645970
+>>>>>>> 2a361fd1b78219385ebf356d98fd63716e645970-->
```

Comment:
There's no need of commenting this out. It will add unnecessary lines to the code. You can just remove this entirely :)